### PR TITLE
homework 1 ready for check

### PR DESCRIPTION
**Pull request by DmitriyMonarkhov**

### CODE REVIEW

File: src/test/java/hw1/TestingClasses/DivideTest.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+public class DivideTest extends SetUpTearDownCalc {
+
+    @Test(
+            dataProviderClass = DivideDP.class,
+            dataProvider = "DP for divide operations"
+    )
+
+    public void divideTest(double numb1, double numb2, double expected) {
+        double actual = calculator.div(numb1, numb2);
```
Comment (ZeroShadows) [inline src/test/java/hw1/TestingClasses/DivideTest.java]: what will happend if numb2 is 0? let's cover this case to

**Review by ZeroShadows — CHANGES_REQUESTED** [review]

**Comment by ZeroShadows** [comment]
general comment - move test classes from folder 'TestClasses' to test folder and remove it

**Comment by DmitriyMonarkhov** [comment]
Corrections and comments completed

File: src/test/java/hw1/dataProviders/DivideDP.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+public class DivideDP {
+
+    @DataProvider(name = "DP for divide operations")
+    public Object[][] divideDP() {
+        return new Object[][] {
+                {34, 2, 17},
+                {8, 8, 1},
+                {5, 2, 2.5},
+                {44, 1, 44},
+                {34, 0, Double.POSITIVE_INFINITY}
```
Comment (ZeroShadows) [inline src/test/java/hw1/dataProviders/DivideDP.java]: what will happend if it -0? 
Take a look here: https://howtodoinjava.com/testng/testng-expected-exception/

Comment (DmitriyMonarkhov) [reply src/test/java/hw1/dataProviders/DivideDP.java]: Thank's! Re-write.

**Review by ZeroShadows — APPROVED** [review]

**Pull request merged**